src/app/workflows/list/list.component.html

```diff
@@ -33,7 +33,7 @@
                 <span class="glyphicon glyphicon-ok" tooltip="Verified"></span>
               </a>
             </span>
-            <a (click)="sendWorkflowInfo(workflow)" [routerLink]="('/workflows/' + workflow.full_workflow_path )">{{ workflow?.repository + (workflow.workflowName ? ' (' + workflow.workflowName + ')' : '') }}</a>
+            <a (click)="sendWorkflowInfo(workflow)" [routerLink]="('/workflows/' + workflow.full_workflow_path )">{{ workflow?.repository + (workflow.workflowName ? '/' + workflow.workflowName : '') }}</a>
```

Comment:
Looks like a mixture of `workflow?.` and `workflow.`
Not sure if this is a problem

Reply:
I think the ? only looks for the .field if the workflow is not null.

Reply:
I think Angular crashes when there isn't a '?' and workflow happens to be null at some point.  If there's no chance of workflow being null, it should be fine.

Reply:
https://stackoverflow.com/questions/45922715/what-is-the-meaning-of-question-mark-in-expressions-in-angular-2